package.json

```diff
@@ -15,8 +14,6 @@
     "babel:modules": "NODE_ENV=production babel ./react/modules --out-dir ./modules --ignore '__tests__,*.spec.js' --ignore '*.md' --copy-files",
     "clean:lib": "rm -rf lib",
     "clean:modules": "rm -rf modules",
-    "compile": "run-s clean:lib clean:modules build:types babel:lib babel:modules",
```

Comment:
doesn't this one need to stay? I guess npm publish uses this one

Reply:
Only used for `prepublishOnly`. Could it be used somewhere else?

Reply:
Reverted.